frontend/apps/racing-car-game/src/features/Game/hooks.ts

```diff
@@ -53,14 +50,14 @@ function useSubscribeSentMessage() {
 
   const getDecodedPayload = (payload: Bytes) => {
     if (meta?.types.others.output) {
-      return meta.createType(meta?.types.others.output, payload).toHuman();
+      return meta.createType(meta?.types.others.output, [null, payload]).toHuman();
```

Comment:
Same as above. Actually, function is repeats itself, can move it to a shared folder.